static/usr/lib/core/handle-writable-paths

```diff
@@ -141,6 +141,29 @@ handle_writable_paths()
 			continue
 		fi
 	done
+
+        # now apply the defaults on top
+        handle_writable_defaults
+}
+
+# handle_writable_default will apply system defaults *once* after
+# the handle_writable_path got populated
+handle_writable_defaults()
+{
+        srcpath="${rootmnt}/writable/system-data/_writable_defaults"
+        if [ ! -d "$srcpath" ] || [ -e "${srcpath}/.done" ]; then
+                return
+        fi
+
+        dstpath="${rootmnt}/writable/system-data"
+        for fileordir in "$srcpath"/* ; do
+            # skip empty $srcpath/
+            [ ! -e "$fileordir" ] && [ ! -L "$fileordir" ] &&  continue
```

Comment:
I guess I don't understand this line of code. We're handling empty srcpath, which -e makes sense, but then if the path does not exist we also check if it is not a symlink? Will the second conditional ever be true when there's an && between?

Reply:
This is to catch the case when there is a broken symlink. In this case we still want to copy the broken symlink (as it may point to something sensible later when the full core20 writable-path stuff is assembled). There is a testcase for this in the new test (that checks if broken symlinks work).

Reply:
Ah, ok, makes sense! Forgot that -e follows symlinks. Thanks for reminding me of that!